it/pom.xml

```diff
@@ -67,6 +68,11 @@
       <artifactId>google-cloud-storage</artifactId>
       <version>${cloud-storage.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-bigquery</artifactId>
+      <version>${google-cloud-bigquery.version}</version>
```

Comment:
Maybe remove this <version> if possible (see comment above)

Reply:
Honestly, I wasn't sure why it needed that dependency entry in the first place when the bigtable RM didn't, but maven was complaining, so I added it. However, it seems to compile fine without the version specified as you suggested.